internal/pkg/testutil/testing.go

```diff
@@ -1,34 +1,38 @@
 package testutil
 
 import (
+	"github.com/segmentio/terraform-docs/internal/pkg/tfconf"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 	"runtime"
-
-	"github.com/segmentio/terraform-docs/internal/pkg/tfconf"
 )
 
-// GetExpected returns 'example' Module and expected Golden file content
-func GetExpected(goldenFile string) (*tfconf.Module, string, error) {
-	path, err := getExampleFolder()
```

Comment:
I wasn't expecting the `GetExpected` function to also generate the module, so I broke it into two separate functions:

1. GetExpected: Read the Golden File and assign to expected
2. GetModule: Accept options and generate the module used for the `actual` var

Reply:
Yes, I've noticed this and have already had it converted to exactly like what you did in #195! Nice work.